libkbfs/kbfs_ops_test.go

```diff
@@ -4919,3 +4919,312 @@ func TestKBFSOpsSyncAllTwoFiles(t *testing.T) {
 		t.Fatalf("Couldn't sync; %+v", err)
 	}
 }
+
+func TestKBFSOpsCreateSyncAll(t *testing.T) {
+	config, _, ctx, cancel := kbfsOpsInitNoMocks(t, "alice", "bob")
+	// TODO: Use kbfsTestShutdownNoMocks.
+	defer kbfsTestShutdownNoMocks(t, config, ctx, cancel)
+
+	// Manually create a file without actually syncing it.
+	// TODO(KBFS-2076) remove all this duplicated code.
+	rootNode := GetRootNodeOrBust(ctx, t, config, "alice,bob", false)
+
+	ops := getOps(config, rootNode.GetFolderBranch().Tlf)
+	name := "myfile"
+	var fileNode Node
+	{
+		rootDir := ops.nodeCache.PathFromNode(rootNode)
+		co, err := newCreateOp(name, rootDir.tailPointer(), File)
+		if err != nil {
```

Comment:
Yeah, but I'm not into mixing `require` and non-`require` in the same test file.  I hope someday we'll have time to convert the whole file at once.